dvc/annotations.py

```diff
@@ -28,9 +28,27 @@ def to_dict(self) -> Dict[str, str]:
 
 
 @dataclass
-class Artifact(Annotation):
+class Artifact:
     PARAM_PATH: ClassVar[str] = "path"
-    path: Optional[str] = None
+    PARAM_DESC: ClassVar[str] = "desc"
+    PARAM_TYPE: ClassVar[str] = "type"
+    PARAM_LABELS: ClassVar[str] = "labels"
+    PARAM_META: ClassVar[str] = "meta"
+
+    path: str
+    desc: Optional[str] = None
+    type: Optional[str] = None  # noqa: A003
+    labels: List[str] = field(default_factory=list)
+    meta: Dict[str, Any] = field(default_factory=dict)
+
+    def update(self, **kwargs) -> "Artifact":
+        for attr, value in kwargs.items():
+            if value and hasattr(self, attr):
+                setattr(self, attr, value)
+        return self
+
+    def to_dict(self) -> Dict[str, str]:
+        return compact(asdict(self))
```

Comment:
Are you using `update()` or `add()` API?

Reply:
No. Deleting it, thanks.